# Laboratorio 7

### Inversiones en Energia, S.A 

Recomendaciones y preguntas:

- Estado de resultados breve del 2017.

- ¿Cómo quedó el tarifario en el 2017por unidad?

- Las tarifas actuales ¿son aceptables por el cliente? ¿Estamos en números rojos?

- ¿Cuándo podríamos perderle a un mantenimiento y/o reparación?

- ¿Debemos abrir más centros de distribución?

- ¿Qué estrategias debo seguir?

Adicionalmente, quiere entenderse visualmente: 

- 80-20 de clientes (postes en este caso)

- Recorridos mas efectivos

In [6]:
import pandas as pd
import numpy as np
import seaborn as sns
print("listo")

listo


In [14]:
data_inicial = pd.read_csv("c1.csv")
data_inicial.head()

Fecha      ID Camion_5     Pickup    Moto                     Cod  \
0  10/12/2017  368224    Q-      Q270.72    Q-     VERIFICACION_MEDIDORES   
1    19-03-17  368224    Q-      Q262.24    Q-     REVISION_TRANSFORMADOR   
2    13-03-17  368224    Q-      Q230.04    Q-                   REVISION   
3    14-04-17  368224    Q-      Q268.77    Q-                   REVISION   
4   11/4/2017  748633    Q-      Q233.33    Q-                   REVISION   

   origen        Lat       Long    factura  ... directoMoto fijoCamion_5  \
0  150277  15.463326 -89.725646   Q316.72   ...       Q-           Q-      
1  150277  15.463326 -89.725646   Q267.24   ...       Q-           Q-      
2  150277  15.463326 -89.725646   Q236.04   ...       Q-           Q-      
3  150224  15.463326 -89.725646   Q289.77   ...       Q-           Q-      
4  150277  14.725681 -90.896437   Q248.33   ...       Q-           Q-      

  fijoPickup fijoMoto height 5-30  30-45 45-75 75-120 120+  
0   Q102.87     Q-         8                            x   
1    Q89.16     Q-         8                            x   
2    Q94.32     Q-         8                            x   
3    Q94.07     Q-         8                            x   
4    Q79.33     Q-         8                            x   

[5 rows x 22 columns]

**Seleccionar los puntos de origen unicos**

In [12]:
origenes = np.unique(data_inicial.origen, axis = 0)
print(origenes)

[150224 150277 150278 150841]


Segun gps-coordinates.org, las coordenadas de cada origen quedan de la siguiente manera:

- 150224 = 
- 150277 = Comalapa
- 150278 = 
- 150841 = 